aries_cloudagent/transport/inbound/manager.py

```diff
@@ -77,8 +77,15 @@ def register(self, config: InboundTransportConfiguration) -> str:
 
         """
         try:
+            if '.' in config.module:
+                package, *mod_parts = config.module.split('.')
```

Comment:
Maybe `package, module = config.module.rsplit(".", 1)`?

Reply:
Thanks for the suggestion!